frontends/precinct-scanner/src/app_root.tsx

```diff
@@ -398,6 +391,7 @@ export function AppRoot({
       precinctSelection: newPrecinctSelection,
     });
     await config.setPrecinctSelection(newPrecinctSelection);
+    await refreshConfig();
```

Comment:
Because we may change other things, like polls state, when changing precinct selection, we need to refresh config after.